src/pbt/prophecy_build_tool.py

```diff
@@ -0,0 +1,472 @@
+import collections
+from glob import glob
+import json
+import os
+import re
+import subprocess
+import sys
+import time
+from typing import Dict
+
+import yaml
+from .process import Process
+from rich import print
+from databricks_cli.configure.config import _get_api_client
```

Comment:
Avoid importing protected members of other libraries

Reply:
I dont think we can NOT use this. any other way?